draft-demarco-oauth-status-assertions.md

```diff
@@ -338,10 +338,10 @@ in the following table.
 | **aud** | It MUST be set with the Issuer Status Assertion endpoint URL as value that identify the intended audience. | {{RFC9126}}, {{RFC7519}} |
 | **exp** | UNIX Timestamp with the expiration time of the JWT. It MUST be superior to the value set for `iat` . | {{RFC9126}}, {{RFC7519}}, {{RFC7515}} |
 | **iat** | UNIX Timestamp with the time of JWT/CWT issuance. | {{RFC9126}}, {{RFC7519}} |
-| **jti** | Unique identifier for the JWT.  | {{RFC7519}} Section 4.1.7 |
-| **cti** | Unique identifier for the CWT.  | {{RFC7519}} Section 4.1.7 |
-| **credential_hash** | Hash value of the Digital Credential the Status Assertion is bound to. | this specification |
-| **credential_hash_alg** | The Algorithm used of hashing the Digital Credential to which the Status Assertion is bound. The value SHOULD be set to `sha-256`. | this specification |
+| **jti** | Unique identifier for the JWT. REQUIRED only if `typ` is `status-assertion-request+jwt`. | {{RFC7519}} Section 4.1.7 |
+| **cti** | Unique identifier for the CWT. REQUIRED only if `typ` is `status-assertion-request+cwt`. | {{RFC7519}} Section 4.1.7 |
```

Comment:
```suggestion
| **jti** | Unique identifier for the JWT. | {{RFC7519}} Section 4.1.7 |
| **cti** | Unique identifier for the CWT. | {{RFC7519}} Section 4.1.7 |
```

We already say that these are the parameter for the status assertion request. It would be beneficial say why jti and cti are useful against replay attacks